conference.js

```diff
@@ -1938,8 +2025,37 @@ export default {
             .then(() => {
                 mediaDeviceHelper.setCurrentMediaDevices(devices);
                 APP.UI.onAvailableDevicesChanged(devices);
+                this.updateVideoIconEnabled();
             });
     },
+    /**
+     * Determines whether or not the video button should be enabled.
+     */
+    updateVideoIconEnabled() {
+        const videoMediaDevices
+            = mediaDeviceHelper.getCurrentMediaDevices().videoinput;
+        const hasPermissions
+            = videoMediaDevices !== undefined
+                && videoMediaDevices.filter(d => d.label !== '').length > 0;
```

Comment:
Thanks for pointing this out ! So currently the button will appear muted in such case, but it should not cause any other side effects, so I guess we can live with that ?